src/views/my-items/MyItemsPage.tsx

```diff
@@ -267,25 +267,31 @@ export default function MyItemsPage() {
           <StyledMenuBar>
             <StyledMenuItem>
               {isMobile ? <SortedIcon /> : <p>{t('sorted_by')}</p>}
-              <Dropdown
-                selected={sortedBy.text}
-                options={sortedByOptions.map(({ text }) => text)}
-                onSelect={text => setSortedBy(sortedByOptions.find(o => o.text === text) || sortedByOptions[0])}
-              />
+              <Caption>
```

Comment:
We should put `<Caption>` in Dropdown.tsx if we want to style dropdown component.

Reply:
Wasn't sure if we wanted all dropdowns to have it. but I can do that too and see how it looks where used

Reply:
@Appppo the Dropdown already has 'SmallContent' on mobile, do we just want to make that even smaller? (14px -> 12px)